cpu/stm32/periph/pm.c

```diff
@@ -40,7 +40,8 @@
  *
  * Available values can be found in reference manual, PWR section, register CR.
  */
-#if defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F3)
+#if defined(CPU_FAM_STM32F0) || defined(CPU_FAM_STM32F3) || \
+    defined(CPU_FAM_STM32F1)
```

Comment:
Just to be sure, this is a merged of the stm32f1-specific code into this file and not an unrelated change?

Reply:
yes, the define was the same in the F1.